src/ServiceManagement/Compute/Commands.ServiceManagement/IaaS/Network/NewAzureIPTag.cs

```diff
@@ -19,7 +19,7 @@
 
 namespace Microsoft.WindowsAzure.Commands.ServiceManagement.IaaS
 {
-    [Cmdlet(VerbsCommon.New, "AzureIPTag"), OutputType(typeof(IPTag))]
+    [Cmdlet(VerbsCommon.New, "AzureIPTag", SupportsShouldProcess = true), OutputType(typeof(IPTag))]
```

Comment:
[Here is some documentation on implementing `ShouldProcess`](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess)